Add periodic update / multiple debug session documentation to README.md (#160)
eclipse-cdt-cloud/vscode-memory-inspector: README.md

3. **Groups per Row**: Number of _Groups_ to display in a row. This can be a fixed number of _Groups_. Or the value `Autofit` to let the Memory Inspector calculate the best utilization of space in the `Data` column.
4. **Group Endianess**: The order of _MAUs_ within a _Group_. The value can be `Little Endian` or `Big Endian`.

![Screenshot of the Memory Inspector](https://raw.githubusercontent.com/eclipse-cdt-cloud/vscode-memory-inspector/main/media/memory-inspector-options-endian-screenshot.png)

The following terminology is used:

- **Byte**: A data unit of 8 Bits.
- **MAU (Minimum Addressable Unit)**: A Minimum Addressable Unit of memory. It consists of one or more _Bytes_ represented by a single address.
- **Group**: A group of _MAUs_ that forms a data value. _Groups_ are the Memory Inspector's default granularity to edit memory contents.
- **Row**: A row in the Memory Inspector display containing multiple _Groups_.
- **Endianess**: The order of displaying _MAUs_ within a _Group_.

### Periodic Updates

> The Memory Inspector can automatically refresh memory data based on different triggers. You can configure the auto-refresh behavior using the following options:

Reviewer: ```suggestion
The Memory Inspector can automatically refresh displayed memory data based on different triggers. You can configure the auto-refresh behavior using the following options:
```

Just to avoid misunderstandings around refreshing memory data in the target system vs memory data in the display.

![Screenshot of the Memory Inspector](./media/memory-inspector-periodic-update-screenshot.png)

> - **On Stop**: Refreshes memory data when the program papuses.

Reviewer: ```suggestion
- **On Stop**: Refreshes memory data when the program pauses.
```
Typo

- **Periodic Refresh**: Refreshes memory data after a specified delay.

>   - **Always**: Refreshes memory data continuously, regardless of the debug session state.

Reviewer: Maybe `...regardless of the program execution state.`

>   - **While Running**: Refreshes memory data only when the debug session is running.

Reviewer: Maybe `...when the program is running.`

>   - **Off**: Disables automatic refreshing of memory data.

Reviewer: ```suggestion
  - **Off**: Disables automatic refresh of memory data.
```

> These settings can be adjusted individually for each Memory Inspector view or in the preferences.

Reviewer: ```suggestion
These settings can be adjusted individually for each Memory Inspector view or globally in the preferences.
```

### Multiple Debug Sessions

![Screenshot of the Memory Inspector](./media/memory-inspector-debug-sessions-screenshot.png)

> The Memory Inspector supports multiple debug sessions. When multiple debug sessions are active, a dropdown will appear in the memory view, allowing you to switch between the sessions. This enables you to target different debug sessions in multiple windows simultaneously.

Reviewer: ```suggestion
The Memory Inspector supports multiple debug sessions. If multiple debug sessions are active, a dropdown appears in the memory view which allows you to switch between them. This enables you to target different debug sessions in multiple windows, simultaneously.
```
